[2025-03-17T11:45:16.700+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: Google_map.transform_data_phase_1_task manual__2025-03-16T19:43:35.857221+00:00 [queued]>
[2025-03-17T11:45:16.734+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: Google_map.transform_data_phase_1_task manual__2025-03-16T19:43:35.857221+00:00 [queued]>
[2025-03-17T11:45:16.735+0000] {taskinstance.py:2170} INFO - Starting attempt 1 of 2
[2025-03-17T11:45:16.807+0000] {taskinstance.py:2191} INFO - Executing <Task(BashOperator): transform_data_phase_1_task> on 2025-03-16 19:43:35.857221+00:00
[2025-03-17T11:45:16.830+0000] {standard_task_runner.py:60} INFO - Started process 25151 to run task
[2025-03-17T11:45:16.845+0000] {standard_task_runner.py:87} INFO - Running: ['airflow', 'tasks', 'run', 'Google_map', 'transform_data_phase_1_task', 'manual__2025-03-16T19:43:35.857221+00:00', '--job-id', '155', '--raw', '--subdir', 'DAGS_FOLDER/google_map_dag_etl.py', '--cfg-path', '/tmp/tmpzcvs34q7']
[2025-03-17T11:45:16.870+0000] {standard_task_runner.py:88} INFO - Job 155: Subtask transform_data_phase_1_task
[2025-03-17T11:45:17.230+0000] {task_command.py:423} INFO - Running <TaskInstance: Google_map.transform_data_phase_1_task manual__2025-03-16T19:43:35.857221+00:00 [running]> on host DESKTOP-AVAHALM.
[2025-03-17T11:45:17.795+0000] {taskinstance.py:2480} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='master_m2si' AIRFLOW_CTX_DAG_ID='Google_map' AIRFLOW_CTX_TASK_ID='transform_data_phase_1_task' AIRFLOW_CTX_EXECUTION_DATE='2025-03-16T19:43:35.857221+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-03-16T19:43:35.857221+00:00'
[2025-03-17T11:45:17.798+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2025-03-17T11:45:17.800+0000] {subprocess.py:75} INFO - Running command: ['/usr/bin/bash', '-c', 'cd ~/data_warehouse_project && dbt run --profiles-dir ~/.dbt --models cleaned_reviews']
[2025-03-17T11:45:17.855+0000] {subprocess.py:86} INFO - Output:
[2025-03-17T11:45:24.814+0000] {subprocess.py:93} INFO - [0m11:45:24  Running with dbt=1.9.0-b2
[2025-03-17T11:45:25.598+0000] {subprocess.py:93} INFO - [0m11:45:25  Registered adapter: postgres=1.8.2
[2025-03-17T11:45:26.288+0000] {subprocess.py:93} INFO - [0m11:45:26  [[33mWARNING[0m]: Configuration paths exist in your dbt_project.yml file which do not apply to any resources.
[2025-03-17T11:45:26.289+0000] {subprocess.py:93} INFO - There are 1 unused configuration paths:
[2025-03-17T11:45:26.289+0000] {subprocess.py:93} INFO - - models.data_warehouse_project.example
[2025-03-17T11:45:27.059+0000] {subprocess.py:93} INFO - [0m11:45:27  Found 1 model, 1 source, 424 macros
[2025-03-17T11:45:27.062+0000] {subprocess.py:93} INFO - [0m11:45:27
[2025-03-17T11:45:27.063+0000] {subprocess.py:93} INFO - [0m11:45:27  Concurrency: 4 threads (target='dev')
[2025-03-17T11:45:27.064+0000] {subprocess.py:93} INFO - [0m11:45:27
[2025-03-17T11:45:27.402+0000] {subprocess.py:93} INFO - [0m11:45:27  1 of 1 START sql table model public.cleaned_reviews ............................ [RUN]
[2025-03-17T11:45:27.897+0000] {subprocess.py:93} INFO - [0m11:45:27  1 of 1 OK created sql table model public.cleaned_reviews ....................... [[32mSELECT 9[0m in 0.49s]
[2025-03-17T11:45:27.978+0000] {subprocess.py:93} INFO - [0m11:45:27
[2025-03-17T11:45:27.980+0000] {subprocess.py:93} INFO - [0m11:45:27  Finished running 1 table model in 0 hours 0 minutes and 0.91 seconds (0.91s).
[2025-03-17T11:45:28.117+0000] {subprocess.py:93} INFO - [0m11:45:28
[2025-03-17T11:45:28.119+0000] {subprocess.py:93} INFO - [0m11:45:28  [32mCompleted successfully[0m
[2025-03-17T11:45:28.120+0000] {subprocess.py:93} INFO - [0m11:45:28
[2025-03-17T11:45:28.122+0000] {subprocess.py:93} INFO - [0m11:45:28  Done. PASS=1 WARN=0 ERROR=0 SKIP=0 TOTAL=1
[2025-03-17T11:45:30.250+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2025-03-17T11:45:30.380+0000] {taskinstance.py:1138} INFO - Marking task as SUCCESS. dag_id=Google_map, task_id=transform_data_phase_1_task, execution_date=20250316T194335, start_date=20250317T114516, end_date=20250317T114530
[2025-03-17T11:45:30.417+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2025-03-17T11:45:30.480+0000] {taskinstance.py:3280} INFO - 1 downstream tasks scheduled from follow-on schedule check
